src/randoop/HelperSequenceCreator.java

```diff
@@ -38,7 +38,7 @@
     } else {
       SimpleList<Sequence> candidates = components.getSequencesForType(cls.getComponentType(), false);
       if (candidates.isEmpty()) {
-        if (GenInputsAbstract.forbid_null) {
+        if (GenInputsAbstract.null_ratio == 0) {
           // No sequences that produce appropriate component values found, and null forbidden.
```

Comment:
It seems that the behavior as specified in the manual, which is to pass a null value as a last resort, is desirable.  I wouldn't want Randoop to give up on creating sequences just because it is unwilling to pass a null value?

(My question "Do you think that Randoop's behavior should be different than what the manual says?" must have been ambiguous, sorry.  It was not intended to be, "Do you think inconsistency is good?", but "Do you disagree with the specification in the manual, and why?")